common/utility.pm

```diff
@@ -1,6 +1,31 @@
+use strict;
 
 package utility;
 
+# Run command, returns 0 if there was an error
```

Comment:
Is the comment accurate? "returns 0 if there was an error"

Reply:
Well sorta. If the command fails the return value depends on the `$ignore_failure` optional argument. Looking at this again, it might be better to move that decision outside the function, but I don't see where the ignore argument is being passed anyway. I would think it would be used based on if the command was "required", but I would have to get back into this PR again to figure out what I was trying to do.